### ISSUE TITLE
False Positive: Add #include <set> for set

### ISSUE THREAD

**Issue by vinograd47** [google/styleguide#410]
I have a template method, named `set`, which is used like this:

```
obj.set<int>("param", value);
```

cpplint assumes that it is a `std::set` usage and requires to include `<set>` header.

**Comment by tkruse** [comment]
Consider submitting the patch to the fork at https://github.com/cpplint/cpplint

**Comment by vinograd47** [comment]
https://github.com/cpplint/cpplint/pull/62

**Comment by vapier** [comment]
public cpplint development in this repository has been halted (see https://github.com/google/styleguide/pull/837). please migrate to https://github.com/cpplint/cpplint now.
